exporters/setup.py

```diff
@@ -1,3 +1,8 @@
 from setuptools import find_packages, setup
 
-setup(name="pelorus", packages=find_packages(where="."), python_requires=">=3.9")
```

Comment:
did not test if this can break source to image yet

Reply:
Why this change in the PR for validation check?

Reply:
I think adding version to exporters can help us do the check